src/components/PeopleManagement/LearnerDetailPage/EnrollmentCard.jsx

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  Badge, Card, Hyperlink, Stack,
+} from '@openedx/paragon';
+import { getConfig } from '@edx/frontend-platform';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+import { camelCaseDict } from '../../../utils';
+
+const EnrollmentCard = ({ enrollment, enterpriseSlug }) => {
+  const courseEnrollment = camelCaseDict(enrollment);
+  const renderBadge = () => {
+    switch (courseEnrollment.courseRunStatus) {
+      case 'completed': {
+        return (<Badge variant="light">Completed</Badge>);
+      }
+      case 'in_progress': {
+        return (<Badge variant="success">In Progress</Badge>);
+      }
+      default: {
+        return (<Badge variant="info">Upcoming</Badge>);
+      }
+    }
+  };
+
+  return (
+    <Card className="mt-4 p-4">
+      <Stack direction="horizontal" className="justify-content-between">
+        <h3 className="mb-1">{courseEnrollment.displayName}</h3>
+        {renderBadge()}
+      </Stack>
+      <p className="small">{courseEnrollment.orgName} • {courseEnrollment.courseType}</p>
+      <Card.Footer className="p-0 justify-content-start">
+        <Hyperlink
+          className="btn btn-outline-primary"
+          target="_blank"
+          destination={`${getConfig().ENTERPRISE_LEARNER_PORTAL_URL}/${enterpriseSlug}/course/${courseEnrollment.courseKey}`}
+        >
+          <FormattedMessage
+            id="adminPortal.peopleManagement.learnerDetailPage.enrollmentCard.learnerPortalLink"
+            defaultMessage="View course"
```

Comment:
Are we going to have a 'View Assignment' button for Assigned courses as in Figma?

Reply:
We talked about this in standup, but for this the endpoint only returns actual enrollments, so for this ticket we don't have the assignment cards.